<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>516</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Penguin SBIG</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox_7">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>0</y>
      <width>191</width>
      <height>371</height>
     </rect>
    </property>
    <property name="title">
     <string>Parametri</string>
    </property>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>171</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string>Binning</string>
     </property>
     <widget class="QComboBox" name="comboBINN">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>20</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="editable">
       <bool>false</bool>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="maxVisibleItems">
       <number>3</number>
      </property>
      <property name="insertPolicy">
       <enum>QComboBox::InsertAtBottom</enum>
      </property>
      <item>
       <property name="text">
        <string>1x1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2x2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3x3</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>171</width>
       <height>51</height>
      </rect>
     </property>
     <property name="title">
      <string>Estensione file</string>
     </property>
     <widget class="QComboBox" name="ImgTypeCB">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>141</width>
        <height>24</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>SBIG</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>FITS</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>171</width>
       <height>51</height>
      </rect>
     </property>
     <property name="title">
      <string>Tipo Frame</string>
     </property>
     <widget class="QComboBox" name="FrameTYPCB">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>141</width>
        <height>24</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Light Frame</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Dark Frame</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Autodark</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>171</width>
       <height>51</height>
      </rect>
     </property>
     <property name="title">
      <string>Tempo Esposizione</string>
     </property>
     <widget class="QLineEdit" name="exposureFld">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>71</width>
        <height>26</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>82</width>
        <height>32767</height>
       </size>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="inputMask">
       <string>009.999; </string>
      </property>
      <property name="text">
       <string>.</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>30</y>
        <width>57</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>sec.</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>300</y>
       <width>81</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string>FR</string>
     </property>
     <widget class="QComboBox" name="comboFR">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>ON</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>OFF</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>300</y>
       <width>91</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string>DCM</string>
     </property>
     <widget class="QComboBox" name="comboDCM">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>ON</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>OFF</string>
       </property>
      </item>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="buttImage">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>380</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>START</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Directory salvataggio:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkSeries">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Serie di </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineNimg">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>70</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="maxLength">
     <number>3</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>105</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Nome Immagini:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>100</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>32773</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Link status:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelStatus">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>150</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>NAN</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_imm">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>261</width>
      <height>211</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="linePath">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>281</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>32773</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>516</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="Exit"/>
   </widget>
   <widget class="QMenu" name="Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="options"/>
   </widget>
   <widget class="QMenu" name="Camera">
    <property name="title">
     <string>&amp;Camera</string>
    </property>
    <addaction name="cameraSetup"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionManuale"/>
    <addaction name="actionSupporto_Tecnico"/>
    <addaction name="separator"/>
    <addaction name="actionAbout_Dev"/>
    <addaction name="actionVersion"/>
   </widget>
   <addaction name="File"/>
   <addaction name="Edit"/>
   <addaction name="Camera"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="Exit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="cameraSetup">
   <property name="text">
    <string>Show Camera Panel</string>
   </property>
   <property name="iconText">
    <string>cameraSetup</string>
   </property>
  </action>
  <action name="options">
   <property name="text">
    <string>Options</string>
   </property>
  </action>
  <action name="actionSupporto_Tecnico">
   <property name="text">
    <string>Supporto Tecnico</string>
   </property>
  </action>
  <action name="actionManuale">
   <property name="text">
    <string>Manuale</string>
   </property>
  </action>
  <action name="actionAbout_Dev">
   <property name="text">
    <string>Sviluppatori</string>
   </property>
  </action>
  <action name="actionVersion">
   <property name="text">
    <string>Version</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>Exit</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>379</x>
     <y>238</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
